zad001.rb

```diff
@@ -0,0 +1,67 @@
+if __FILE__ == $0
+
+  # Tresc zadania:
+  # In the above example, the sum of each three-measurement window is as follows:
+
+# A: 607 (N/A - no previous sum)
+# B: 618 (increased)
+# C: 618 (no change)
+# D: 617 (decreased)
+# E: 647 (increased)
+# F: 716 (increased)
+# G: 769 (increased)
+# H: 792 (increased)
+
+# In this example, there are 5 sums that are larger than the previous sum.
+
+# Consider sums of a three-measurement sliding window. How many sums are larger than the previous sum?
+# Koniec tresci. -----------------------------------------------------------------------------------
+
+
+  puts (' Type filename with extension .txt (or type EXIT if you want to quit) end press ENTER :')
+  filee = gets # .to_s # roznica gets, STDIN.gets $stdin.gets
```

Comment:
`filee` variable is defined but not used

Reply:
yes, but it schould be used we need to discuss :)

Reply:
@kc-si here is an example in irb:

```
irb(main):001:0> user_input = gets
numbers.txt
=> "numbers.txt\n"
irb(main):002:0> filename = user_input.chop
=> "numbers.txt"
irb(main):003:0> file_content = File.read(filename)
=> "12345\n43\n26\n453\n234.35\n"
```

As you can see, an user input from STDIN always have new line at the end and you need to remove it to get a pure value from the user (in this case the filename). You can remove last character from the string using https://ruby-doc.org/core-3.1.0/String.html#method-i-chop method.

In your code, you could use it this way:

```ruby
filename = gets.chop
File.read(filename) # ....
```

Reply:
ok, probably this is what I need, last character was a problem I think